rider/views.py

```diff
@@ -133,9 +133,9 @@ def check_code_view(request, aff_id):
 
 def get_callback_from_request(request):
     if 'GET' == request.method:
-        return request.GET[u'callback']
+        return request.GET.get('callback', None)
```

Comment:
:+1: Does it pay to also fix [this one](https://github.com/TourTrak/devcycle-server/blob/staging/location_update/views.py#L292)?